README.md

```diff
@@ -62,6 +62,7 @@ Name|Type|Default|Description
 `quotesOnKeys`|`boolean`|`true`|set to false to remove quotes from keys (eg. `"name":` vs. `name:`)
 `validationMessage`|`string`|"Validation Error"|Custom message for validation failures to `onEdit`, `onAdd`, or `onDelete` callbacks
 `displayArrayKey`|`boolean`|`true`|When set to `true`, the index of the elements prefix values
+`objectCollpasedMaxPropsCount`|`boolean` or `intiger`|`false`|When set to `true`, see object props when object is collpased, intiger set max display props count
```

Comment:
Hi, is that typo about "object**Collpased**MaxPropsCount" and **`intiger`**

Reply:
yes, thanks